ng2-components/ng2-alfresco-core/src/services/notification.service.spec.ts

```diff
@@ -43,33 +48,29 @@ describe('NotificationService', () => {
         fixture.detectChanges();
     });
 
-    describe('openSnackMessage', () => {
 
-        it('should open a message notification bar', (done) => {
-            let promise = fixture.componentInstance.sendMessage();
-            promise.afterDismissed().subscribe(() => {
-                done();
-            });
+    it('should open a message notification bar', (done) => {
+        let promise = fixture.componentInstance.sendMessage();
+        promise.afterDismissed().subscribe(() => {
+            done();
+        });
 
-            fixture.detectChanges();
+        fixture.detectChanges();
 
-            expect(document.querySelector('snack-bar-container')).not.toBeNull();
-        });
+        expect(document.querySelector('snack-bar-container')).not.toBeNull();
     });
 
-    describe('openSnackMessageAction', () => {
 
-        it('should open a message notification bar with action', (done) => {
-            let promise = fixture.componentInstance.sendMessageAction();
-            promise.afterDismissed().subscribe(() => {
-                done();
-            });
```

Comment:
I tried several times, but every experiment requires pushing content to Travis and waiting for results. All tests work fine on local machine.